src/Forms/Prism.Forms/Navigation/IInitializeAsync.cs

```diff
@@ -2,8 +2,16 @@
 
 namespace Prism.Navigation
 {
+    /// <summary>
+    /// Asynchronous initialization of navigation
```

Comment:
No need to add "asynchronous" in the description. This class is used to initialize views and viewmodels during the navigation process

Reply:
done